### PR TITLE
Refactor: use markdown

### PR DESCRIPTION
**Pull request by marcoscaceres**
Switched the document to use markdown, as it will make it easier to maintain. 

Also fixed up markup usage and various ReSpec configuration things.

### CODE REVIEW

**Comment by torgo** [comment]
Hi Marcos - we're just taking a look at this. Is this a statement about the doc itself or a statement about how it fits into the wider ecosystem of docs? I'm trying to figure out in what way you think this will be easier to maintain? Happy to move in this direction but also need to get @hadleybeeman's feedback.

**Comment by marcoscaceres** [comment]
Hi @torgo, thanks for the update. Note in case it's not clear, I only changed the markup to markdown, not the actual content (except for a tiny typo). 

A question that I have is about exporting the principles so other specs can refer to them.

For example, maybe we should:

```
The web must enable <dfn id="expression" data-export="">freedom of expression</dfn>
```

So then other specs can refer to this, like `Supports [=freedom of expression=]!`.

**Review by torgo — APPROVED** [review]
lgtm

File: index.html [hunk 1]
```diff
@@ -196,14 +127,16 @@ <h2>Principles</h2>
 <a data-cite="design-principles#priority-of-constituencies">priority of constituencies</a>.
 We will build new web technologies in a collaborative matter
 according to open processes
-(for example, the <a href="https://www.w3.org/Consortium/Process/">W3C process</a>).
+(for example, the [W3C process](https://www.w3.org/Consortium/Process/)
```
Comment (jyasskin) [inline index.html:130]: This wound up broken at https://w3ctag.github.io/ethical-web-principles/#noharm.

Comment (marcoscaceres) [reply index.html:130]: you are right. Having a look.

Comment (marcoscaceres) [reply index.html:130]: Oh right, I forgot! GitHub flavored (maybe all?) markdown doesn't work inside a html tag. In this case, a `<dd>`.

Comment (marcoscaceres) [reply index.html:130]: https://github.com/w3ctag/ethical-web-principles/pull/43

**Pull request merged**